src/components/Drawer.js

```diff
@@ -99,6 +93,7 @@ const Drawer = React.createClass({
   _animateComponent (transition, extraOptions) {
     const el = this._component;
     const options = Object.assign({
+      delay: 50,
```

Comment:
Is this going to need to be longer in wrapped web views or mobile browsers with slower javascript execution?